.github/workflows/publish-ext-python-apidocs.yml

```diff
@@ -0,0 +1,61 @@
+name: Publish Python API Docs
+
+# Run when the Python API changes or every month so that the artifact does not expire
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - docs/extensions/**
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+concurrency:
+  group: "apidocs-python"
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    name: Generate Python API docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install tools
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3-pip
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install -r requirements-dev.txt
+          python3 -m pip install onnxruntime>=1.12.0
+          python3 -m pip install transformers
+          python3 -m pip install tensorflow_text
+          python3 -m pip install requests
+          python3 -m pip install --index-url https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly/PyPI/onnxruntime-extensions/ onnxruntime-extensions
```

Comment:
This won't work.  You also need to add "--no-index". 
See: https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/run_python_tests.sh#L46

Reply:
got it, thanks!